CollapseLauncher/Classes/InstallManagement/BaseClass/InstallManagerBase.cs

```diff
@@ -336,8 +338,10 @@ protected virtual async ValueTask<int> ConfirmDeltaPatchDialog(DeltaPatchPropert
                         await _gameRepairTool!.StartRepairRoutine(true)!;
                     }
                 }
-                catch
+                catch (Exception ex)
+
                 {
+                    await SentryHelper.ExceptionHandlerAsync(ex, SentryHelper.ExceptionType.UnhandledOther);
                     IsRunning = false;
                     throw;
```

Comment:
This catch might probably throw ``OperationCancelledException`` or ``TaskCancelledException`` too due to the usage of Game Repair mechanism for checking the game integrity before the delta-patch come into play.